lib/src/code_field/code_field.dart

```diff
@@ -325,32 +325,63 @@ class CodeFieldState extends State<CodeField> {
     );
   }
 
-  void _updateCursorOffset(String text) {
-    TextPainter painter = TextPainter(
+  void _updatePopupOffset(String text) {
+    final TextPainter textPainter = _getTextPainter(text);
+    final caretHeight = _getCaretHeight(textPainter);
+
+    final double normalLeftOffset = _getPopupLeftOffset(textPainter);
+    final double normalTopOffset = _getPopupTopOffset(textPainter, caretHeight);
+    final double flippedLeftOffset =
+        // TODO(nausharipov): find where 100 comes from
+        windowSize.width - Sizes.autocompletePopupMaxWidth - 100;
```

Comment:
Should not the left offset be the same for normal and flipped popups?

Reply:
<img width="211" alt="Screen Shot 2022-09-13 at 15 24 13" src="https://user-images.githubusercontent.com/31556582/189864678-ce3f2181-44f7-4081-911c-0d01e53db09f.png">
<img width="399" alt="Screen Shot 2022-09-13 at 15 23 33" src="https://user-images.githubusercontent.com/31556582/189864694-6579aebb-e1b6-4d9f-a4dc-87a663c04f81.png">

Reply:
1) If it is the same as normal.
2) If it is limited.